docs/pages/x/api/date-pickers/date-range-picker.json

```diff
@@ -55,7 +55,6 @@
     "onError": { "type": { "name": "func" } },
     "onMonthChange": { "type": { "name": "func" } },
     "onOpen": { "type": { "name": "func" } },
-    "onViewChange": { "type": { "name": "func" } },
```

Comment:
This prop is never used (and drop on the new pickers)
I can add it back to the typing manually but I don't think it's worth it

Same on other legacy range pickers